src/Symfony/Component/Mime/Email.php

```diff
@@ -58,9 +58,9 @@ class Email extends Message
     /**
      * @return $this
      */
-    public function subject(string $subject): static
+    public function subject(string|\Stringable $subject): static
```

Comment:
This may be a breaking change for apps that extend this class and override this method (https://3v4l.org/uuJj9)

Reply:
Supporting a TranslatableInterface is absolutely not the same than supporting Stringable. There is no requirement for TranslatableInterface to be Stringable.

Reply:
Yeah, my idea was off the cuff. Needs to be thought out more.

Reply:
btw, even for the core TranslatableMessage (that is stringable), casting it to string does not translate it. It returns the translation key.

Reply:
Right, I thought that'd fine as that's the behavior of using keys that don't have a translation.

Reply:
Listener checks for TranslatableMessage, not the TranslatableInterface

Reply:
My thinking was to allow the subject to be *any* stringable object allowing a listener to intercept. If none does, it would just be cast to a string.

Reply:
Adding support for Stringable and then doing weird magic to support TranslatableMessage while the changelog says it adds support for translatable subject (which would be understood as being the interface by many people) looks wrong to me (and limiting as there are valid use cases for other implementations of TranslatableInterface)